### PR TITLE
Renaming `flkl` to `rlkl` and updating docs

### PR DESCRIPTION
**Pull request by marcospb19**

### CODE REVIEW

**Comment by marcospb19** [comment]
I did a lot of changes in the documentation side of it, while also trying to manually synchronise `README.md` and `lib.rs`.
@GrayJack can you review?

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,14 +17,14 @@
 
 ## 1.1.0
  - Add for all comprehensions a Implicit-`for` (or Just-`in`) version
- - Add new version of `lkl` and `flkl` that works almost like array init syntax (`[$init_elem; $n_times]`)
+ - Add new version of `lkl` and `lklk` (old `rlkl`) that works almost like array init syntax (`[$init_elem; $n_times]`)
```
Comment (GrayJack) [inline CHANGELOG.md]: Here and in the line 27 you changed the name of `flkl` to `lklk`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Changes
 
 ## Unreleased
- - Change the behaviour of the `cow!` macro
```
Comment (GrayJack) [inline CHANGELOG.md:4]: You're deleting another unrelated item in the changelog

**Review by GrayJack — COMMENTED** [review]
Awesome PR!!

Just 2 comments

**Review by GrayJack — APPROVED** [review]

**Pull request merged**